README.md

```diff
@@ -33,45 +32,46 @@ Binary downloads are available here: [releases](https://github.com/IBM-Bluemix/g
 
 ## Installation
 
-No special steps are required for installation. See [Installing plug-ins in UrbanCode Deploy](https://developer.ibm.com/urbancode/docs/installing-plugins-ucd/).
+See [Installing plug-ins in UrbanCode Deploy](https://developer.ibm.com/urbancode/docs/installing-plugins-ucd/) and follow the steps for installing an automation plug-in.
 
 ## Usage
 
-### SetupBluemix 
+The IBM Globalization Pipeline plug-in for UrbanCode Deploy contains two processes that need to be configured.
+
+### SetupBluemixV2
 
 #### Description
 
-This step will look for the Globalization Pipeline service instance in your space and will create one if none existing. 
-Tf existing many instances already, the instance contains UCD in its name will take prior priority.
+This process is for configuring your Bluemix account parameters.  When configured, the Globalization Pipeline plug-in will search your Bluemix account in the organization and space that you specify for any instances of the Globalization Pipeline service.  If the Globalization Pipeline service is not found, an instance of the service will be created automatically. 
+If an instance is found that contains *UCD* in its name, that service instance will be given priority.
 
 #### Parameters
 
-Fill in the * fileds according to the Bluemix page
+Required fields are denoted by an * on the UrbanCode Deploy properties configuration page:
 
-* **Name** could be any name you like.
-* **Bluemix UserID** is the login id of the Bluemix page.
-* **Bluemix Password** is the login password of the Bluemix page.
-* **Bluemix Space** is the space of the Bluemix page.
-* **Bluemix Org** is the org of the Bluemix page.
-* **Bluemix API Endpoint** is the url of bluemix API endpoint.  
-   For eg: *https://api.stage1.ng.bluemix.net*
+* **Name** A unique name for the service.
+* **Bluemix UserID** The Bluemix ID for your account.
+* **Bluemix Password** The Bluemix password for your account.
+* **Bluemix Space** The space where you would like to create the Globalization Pipeline service instance.
+* **Bluemix Org** The organization where you would like to create the Globalization Pipeline service instance.
+* **Bluemix API Endpoint** The URL of a Bluemix API endpoint.  
+   For example: *https://api.stage1.ng.bluemix.net*
 
-### Translate 
+### TranslateV2

```

Comment:
That is correct, and the same applies to the first step as well, SetupBluemixV2.  I am assuming that this would be correct because these are the names that show up in the UCD UI when the plug-in is installed.  But we can certainly change them if this is not the case.